docs/site/todo-list-tutorial-repository.md

```diff
@@ -0,0 +1,77 @@
+---
+lang: en
+title: 'Add TodoList Repository'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/todo-list-tutorial-repository.html
+summary: LoopBack 4 TodoList Application Tutorial - Add TodoList Repository
+---
+
+### Repositories with related models
+
+The repository of a model with relations can make constrained version of the
```

Comment:
The first line is very confusing and hard to understand IMO for someone reading it for the first time. Also it would be good to explain what exactly is meant by a `constrained repository`? (You know it, I know it but someone reading it for the first time will be lost).